test/integ/cli/test_cli_incremental_download.py

```diff
@@ -0,0 +1,501 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+Integration tests for the incremental download CLI functionality.
+"""
+
+import os
+import subprocess
+import time
+from typing import Optional, Tuple
+
+import boto3
+import pytest
+
+from deadline.client.api._job_monitoring import wait_for_job_completion
+from .job_templates import (
+    submit_dep_chain_job,
+    submit_dep_data_flow_job,
+    submit_make_many_small_files_job,
+)
+from .test_utils import DeadlineCliTest
+
+
+class IncrementalDownloadTest:
+    """
+    Class for Incremental Download Integration Tests.
+    """
+
+    def __init__(self, farm_id: str, queue_id: str):
+        self.farm_id = farm_id
+        self.queue_id = queue_id
+        self.deadline_client = boto3.client("deadline")
+
+    def wait_for_job_completion(
+        self, job_id: str, timeout: int = 600, poll_interval: int = 5
+    ) -> Tuple[bool, str]:
+        """Wait for a job to complete. Returns (isSuccess, final_status)."""
+        try:
+            result = wait_for_job_completion(
+                farm_id=self.farm_id,
+                queue_id=self.queue_id,
+                job_id=job_id,
+                timeout=timeout,
+                max_poll_interval=poll_interval,
+            )
+            return result.status == "SUCCEEDED", result.status
+        except Exception:
+            return False, "TIMEOUT"
+
+    def run_incremental_download_without_storage_profiles(
+        self,
+        output_dir: str,
+        force_bootstrap: bool = False,
+        conflict_resolution: Optional[str] = None,
+        lookback_window: Optional[int] = None,
+        test_name: Optional[str] = None,
+    ) -> subprocess.CompletedProcess:
+        """Run the incremental download CLI command."""
+        # Use test-specific checkpoint directory to avoid conflicts between parallel tests
+        checkpoint_suffix = f"_{test_name}" if test_name else ""
```

Comment:
nit: Will this be sufficient for parametrized tests? Maybe we can add a random string to the end?

or maybe we can just make sure to always pass in the full test name that pytest collects for these cases since it will include parameters in the test name for parametrized tests like `test_something[happy-path]`, `test_something[sad_path]`

.. we can do this later if/when we have parametrized tests